engine/execution/ingestion/core.go

```diff
@@ -61,22 +60,6 @@ type Core struct {
 	eventConsumer     EventConsumer
 }
 
-// Throttle is used to throttle the blocks to be added to the processables channel
```

Comment:
Moving the interface definition to its implementation, easier to find its implementation.